pkgs/applications/science/logic/klee/default.nix

```diff
@@ -0,0 +1,62 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, cmake
+, llvmPackages_9
+, clang_9
+, python3
+, zlib
+, z3
+, stp
+, cryptominisat
+, gperftools
+, sqlite
+, gtest
+, lit
+, debug ? false
+}:
+
+let
+  kleePython = python3.withPackages (ps: with ps; [ tabulate ]);
+in
+stdenv.mkDerivation rec {
+  pname = "klee";
+  version = "2.2";
+  src = fetchFromGitHub {
+    owner = "klee";
+    repo = "klee";
+    rev = "v${version}";
+    sha256 = "Ar3BKfADjJvvP0dI9+x/l3RDs8ncx4jmO7ol4MgOr4M=";
+  };
+  buildInputs = [ llvmPackages_9.llvm clang_9 z3 stp cryptominisat gperftools sqlite ];
+  nativeBuildInputs = [
+    cmake
+    gtest
+
+    # Should appear BEFORE lit, since lit passes through python rather than the python environment we make.
+    kleePython
+    (lit.override { python3 = kleePython; })
+  ];
+  cmakeFlags = [
+    "-DCMAKE_BUILD_TYPE=${if debug then "Debug" else "Release"}"
+    "-DKLEE_RUNTIME_BUILD_TYPE=${if debug then "Debug" else "Release"}"
+
+    # FIXME: Appears to be broken, similar to this issue. Unclear how to fix it.
+    # https://github.com/google/googletest/issues/2157
+    "-DENABLE_UNIT_TESTS=OFF"
```

Comment:
Put a message out to their mailing list about this issue. The system tests work, that's what matters